RiotSwiftUI/Modules/UserSessions/UserOtherSessions/View/UserOtherSessions.swift

```diff
@@ -87,6 +88,7 @@ struct UserOtherSessions: View {
                 }
                 .background(theme.colors.background)
             }
+            .accessibilityIdentifier(VectorL10n.userOtherSessionClearFilter)
```

Comment:
It's not a good idea to set localizable identifier. We tried to avoid it before.

Reply:
Ok, could you explain what's the problem with this approach?